### PR TITLE
ticdc: add cdc summary monitor

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)
add CDC summary monitor

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/13331
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-summary-monitor.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+- **Full Flush Duration**: the total duration consumed by each transaction in TiCDC from sorting by Sorter to sending downstream. By subtracting the **Backend Flush Duration** value from this value, you can get the total queueing time of a transaction before it is executed on the downstream. If the queueing time is too long, you can consider allocating more memory quota to the replication task.
+
+## MQ Sink panel
+
+The **MQ Sink** panel only has data when the downstream is Kafka.
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-mq-sink.png)
+
+- **Worker Send Message Duration Percentile**: The delay of TiCDC MQ Sink worker sending data downstream.
+- **Kafka Ongoing Bytes**: The rate at which TiCDC MQ Sink sends data downstream.
```
Comment (qiancai) [inline ticdc/ticdc-summary-monitor.md:85]:
```suggestion
- **Kafka Ongoing Bytes**: The speed at which TiCDC MQ Sink sends data to the downstream.
```

File: ticdc/ticdc-summary-monitor.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+## Cloud Storage Sink panel
+
+The **Cloud Storage Sink** panel only has data when the downstream is Cloud Storage.
```
Comment (qiancai) [inline ticdc/ticdc-summary-monitor.md:89]:
```suggestion
The **Cloud Storage Sink** panel displays data only when the downstream is Cloud Storage.
```

File: ticdc/ticdc-summary-monitor.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-cloud-storage.png)
+
+- **Write Bytes/s**: The rate at which the Cloud Storage Sink module writes data to the downstream.
+- **File Count**: the total number of files written by the Cloud Storage Sink module.
+
+## Redo panel
+
+The **Redo** panel only has data when the Redo Log feature is enabled.
```
Comment (qiancai) [inline ticdc/ticdc-summary-monitor.md:98]:
```suggestion
The **Redo** panel displays data only when the Redo Log feature is enabled.
```

File: ticdc/ticdc-summary-monitor.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+![TiCDC Summary Dashboard - Transaction Sink metrics](/media/ticdc/ticdc-summary-monitor-redo.png)
+
+- **Redo Write rows/s**: the number of rows written per second by the Redo module. When the Redo feature is enabled, if the delay of a replication task increases, you can observe whether there is a significant difference between this metric and the value of Puller Output event/s. If so, it might be due to the insufficient writing capacity of the Redo module causing an increase in delay.
```
Comment (qiancai) [inline ticdc/ticdc-summary-monitor.md:102]:
```suggestion
- **Redo Write rows/s**: the number of rows written per second by the Redo module. When the Redo feature is enabled, if the latency of a replication task increases, you can observe whether there is a significant difference between this metric and the value of Puller Output event/s. If so, the increase in latency might be due to the insufficient writing capacity of the Redo module.
```

File: ticdc/ticdc-summary-monitor.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+- **Redo Write Byte/s**: the rate at which data is written per second by the Redo module.
```
Comment (qiancai) [inline ticdc/ticdc-summary-monitor.md:103]:
```suggestion
- **Redo Write Byte/s**: the speed at which data is written per second by the Redo module.
```

File: ticdc/ticdc-summary-monitor.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+Starting from v7.0.0, when you deploy Grafana using TiUP, the TiCDC Summary Dashboard is automatically added to the Grafana monitoring page. Through this dashboard, you can quickly understand the status of TiCDC servers and changefeeds.
+
+The following image shows the monitoring panels of the TiCDC Summary Dashboard:
+
+![TiCDC Summary Dashboard - Overview](/media/ticdc/ticdc-summary-monitor.png)
+
+Each monitoring panel is described as follows:
+
+- Server: summary of TiCDC nodes in the cluster.
+- Changefeed: the delay and status information of TiCDC changefeeds.
```
[the remark this answers is not in the document]
Comment (qiancai) [reply ticdc/ticdc-summary-monitor.md:17]: both "latency" and "delay" are used in this doc

File: ticdc/ticdc-summary-monitor.md (hunk 1 above)
Comment (qiancai) [reply ticdc/ticdc-summary-monitor.md:85]: both "speed" and "rate" are used in this doc

File: TOC.md [hunk 7]
```diff
@@ -526,7 +526,8 @@
       - [Log Filter](/ticdc/ticdc-filter.md)
       - [Bidirectional Replication](/ticdc/ticdc-bidirectional-replication.md)
     - Monitor and Alert
-      - [Monitoring Metrics](/ticdc/monitor-ticdc.md)
+      - [Monitoring Metrics Summary](/ticdc/ticdc-summary-monitor.md)
+      - [Monitoring Metrics Details](/ticdc/monitor-ticdc.md)
```
[the remark this answers is not in the document]
Comment (qiancai) [reply TOC.md:530]: LGTM

**Review by qiancai — APPROVED** [review]
other LGTM

**Review by asddongmen — APPROVED** [review]

**Pull request merged**